packages/account/src/Sections/Profile/PhoneVerification/phone-verification-page.tsx

```diff
@@ -7,18 +7,51 @@ import ConfirmPhoneNumber from './confirm-phone-number';
 import OTPVerification from './otp-verification';
 import CancelPhoneVerificationModal from './cancel-phone-verification-modal';
 import VerificationLinkExpiredModal from './verification-link-expired-modal';
+import { observer, useStore } from '@deriv/stores';
+import { useSendOTPVerificationCode } from '@deriv/hooks';
+import { Loading } from '@deriv/components';
```

Comment:
How is the design for loading component? should'nt we show quill-ui Loading component ?